### PR TITLE
div. Bugs and ERMS:3187

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: server/gokbg3/grails-app/controllers/org/gokb/PublicController.groovy [hunk 1]
```diff
@@ -102,7 +104,14 @@ class PublicController {
...
 
     result =  ESSearchService.search(mutableParams)
-    result.transforms = grailsApplication.config.packageTransforms
+
+    Calendar calendar = Calendar.getInstance()
+    result.componentsOfStatistic = ["TitleInstance", "Org", "Package", "Platform", "CuratoryGroup", "TitleInstancePackagePlatform"]
+
+    result.countComponent = [:]
+    result.componentsOfStatistic.each {String component ->
+      result.countComponent."${component}" = ComponentStatistic.executeQuery("select numTotal from ComponentStatistic where componentType = ? and year = ? and month = ?", [component, calendar.get(Calendar.YEAR), calendar.get(Calendar.MONTH)], [readOnly: true])[0]
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/controllers/org/gokb/PublicController.groovy]: Using ? as queryParam is actually deprecated and against LAS:eR coding convention

File: server/gokbg3/grails-app/views/home/showRules.gsp [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <meta name="layout" content="main"/>
     <asset:stylesheet src="gokb/application.css"/>
 <asset:javascript src="gokb/application.js" />
-    <title>GOKbo : Show Rules</title>
+    <title><g:message code="gokb.appname" default="we:kb"/>o : Show Rules</title>
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/views/home/showRules.gsp]: we:kbo?

**Review by croqueGrec09 — COMMENTED** [review]
two remarks

**Comment by Moetez16** [comment]
Geupdatet!

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**